packages/documentation/pages/components/form-fields.vue

```diff
@@ -279,6 +285,7 @@ const components: Array<{
 	formKey: string
 	name: string
 	supports: Kotti.Field.Supports
+	slots?: Array<string>
```

Comment:
a bit confused here 

is it not the slots object that has name ad content here

Reply:
ah no, it's an array of strings